package.json

```diff
@@ -35,12 +36,15 @@
     "grunt-contrib-copy": "1.0.0",
     "grunt-contrib-jasmine": "1.0.3",
     "grunt-contrib-watch": "1.0.0",
-    "grunt-rollup": "1.0.1",
+    "grunt-rollup": "6.0.0",
```

Comment:
I updated build related packages in hopes we will gain some size improvements. But in the end it didn't help. 
Nevertheless it works fine with update.